Modelica/Electrical/Polyphase/Sensors/AronSensor.mo

```diff
@@ -69,17 +69,19 @@ equation
-<p>
-Contains two 
-<a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a>
-to measure total active power in a three-phase system.
-</p>
-<p>
-This device works only for sinusoidal voltages and currents in three-phase systems without neutral.
-</p>
-<p>
-Elmar Schr&uuml;fer, Leonhard Reindl, Bernhard Zagar: Elektrische Messtechnik. Carl Hanser Verlag.<br>
-Reinhard Lerch: Elektrische Messtechnik. Springer Vieweg.
-</p>
+<p>Contains two <a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a> to measure total active power in a three-phase system. </p>
+<p>This device works only for in three-phase systems without neutral. </p>
+<p><br>The rationale behind this sensor is that power exchanged between two parts of a circuit is the sum of the products of currents in the <i>m</i> wires connecting the two parts times wires&apos; potentials evaluated with reference to an arbitrary potential <i>v</i><sub>ref</sub>: </p>
+<p><i>P</i>=(<i>v</i><sub>1</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>1</sub>+(<i>v</i><sub>2</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>2</sub>+... (<i>v</i><sub>m</sub>-<i>v</i><sub>ref</sub>)*<i>i</i><sub>m</sub></p>
+<p>In case of a three-phase system without neutral we may want to measure power flowing in a line, connecting say the left part of the circuit to its right part. The wires connecting the two parts are only three (i.e. <i>m</i>=3)
+ and therefore all currents between these two parts are <i>i</i><sub>1</sub>, <i>i</i><sub>2</sub>, <i>i</i><sub>3</sub>.&nbsp;</p>
+<p>Since the voltage to be taken as reference is arbitrary, we can take the voltage of conductor 2. Therefore our power becomes:</p>
+<p><i>P</i>=(<i>v</i><sub>1</sub>-<i>v</i><sub>2</sub>)*<i>i</i><sub>1</sub>+(<i>v</i><sub>2</sub>-<i>v</i><sub>2</sub>)*<i>i</i><sub>2</sub>+(<i>v</i><sub>3</sub>-<i>v</i><sub>2)</sub>*<i>i</i><sub>3 </sub>= 
+(<i>v</i><sub>1</sub>-<i>v</i><sub>2</sub>)*i<sub>1</sub>+(<i>v</i><sub>3</sub>-<i>v</i><sub>2</sub>)* i<sub>3</sub></p>
+<p>In this way, we can just sum up the power from two wattmeters to get the three-phase power.</p>
+<p>Note that this formula does not work if there are additional current paths between the left and right parts of our circuits, e.g. if both have grounds (and current flows through it).</p>
+<p>-------------</p>
+<p>For more info on why power flowing in a circuit between two subcircuits is the sum pf products of voltages times currents, the voltages being measured to n arbitrary reference potential, see </p>
+<p>M. Ceraolo and D. Poli; <i>Fundamentals of Electric Power Engineering</i>, par. 3.8.1, IEEE/Wiley, ISBN 978-1-118-67969-2. <a href=\"https://www.wiley.com/en-us/Fundamentals+of+Electric+Power+Engineering:+From+Electromagnetics+to+Power+Systems-p-9781118679692\">https://www.wiley.com/en-us/Fundamentals+of+Electric+Power+Engineering:+From+Electromagnetics+to+Power+Systems-p-9781118679692</a></p>
```

Comment:
@christiankral Should the reference style be updated as you did for Analog and HeatFlow?

Reply:
> @christiankral Should the reference style be updated as you did for Analog and HeatFlow?

@max-privato Yes indeed, the reference should be in the same style as Analog and HeatFlow

Reply:
@max-privato Please also add the publication year

Reply:
I don't know the exact position where you intend to have the year. 
I'll try one position. In case it is not the one you like, please tell me where I can find an example considered to be good for MSL references.

Reply:
The position of the year is fine as you put it. However, I have to update `Modelica.UsersGuide.Conventions.UsersGuide.References` to add, that the DOI and the ISBN number shall be put at the end of the bibliographic data.

Reply:
If I understand well, no further action is required from me.  Thank you for advising me on this reference, and making the final fix.